app/views/homepage/_government_activity.erb

```diff
@@ -0,0 +1,71 @@
+<!--
+Created a custom section and header pattern to
+- display a border-top of 2px (to replace .home-divider)
+- margin-bottom of 30px (both mobile and desktop)
+- padding-top to create a gap between border and heading
+-->
+<section class="homepage-section homepage-section__government_activity">
+  <div class="govuk-grid-row home-numbers-and-info">
+    <div class="govuk-grid-column-two-thirds" data-module="gem-track-click">
+      <div class="homepage-section__heading">
+        <%= render "govuk_publishing_components/components/heading", {
+          text: t('homepage.index.government_activity'),
```

Comment:
Let's use double quotes here:
```suggestion
          text: t("homepage.index.government_activity"),
```

Reply:
Updated and for all other occurrences in the view.